EIPS/eip-1337.md

```diff
@@ -0,0 +1,262 @@
+---
+eip: 1337
+title: [WIP] Subscriptions on the blockchain.
+author: 
```

Comment:
Please format this in a single line, comma-separated, with either username or email address (not both)